### PR TITLE
Small UI Improvements

### PR DESCRIPTION
**Pull request by hassancodess**
## Changes proposed

- Added smooth transitions to Navbar buttons 
- Changed Navbar button style
- Added smooth transitiosn to Form button
- Changed input placeholder text size
- Changed submit button style

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

N/A

## Note to reviewers

N/A

### CODE REVIEW

File: src/components/form.jsx [hunk 1]
```diff
@@ -77,15 +77,15 @@ const Form = () => {
...
+                className="flex-1 w-1/2 h-11 rounded-full ml-2 border-none outline-none text-gray text-lg bg-ash placeholder:text-base"
                 value={userInput}
                 onChange={(e) => setUserInput(e.target.value.toLocaleLowerCase())}
               />
             </div>
 
             <button
               onClick={fetchData}
-              className="bg-purple text-ash font-bold rounded-xl hover:scale-110 p-2 mt-2 md:mt-0"
+              className="bg-purple text-ash font-bold rounded-xl hover:scale-105 py-2 px-3 mt-2 md:mt-0 duration-300 ease-in-out transition"
```
Comment (Njong392) [inline src/components/form.jsx:88]: Nice transition! Thank you for this :tada:

File: src/components/navbar.jsx [hunk 2]
```diff
@@ -10,8 +10,8 @@ const Navbar = () => {
...
+                    <Link to="form" smooth={true} duration={600} className='bg-purple text-ash font-bold rounded-xl py-2 px-5 hover:bg-purpleHover hover:scale-105 outline-none hidden md:block ml-4 cursor-pointer duration-300 ease-in-out'>Get Started </Link>
```
Comment (Njong392) [inline src/components/navbar.jsx:14]: When I check the vercel preview link, the hover states you applied to these are not effective. Perhaps you could take a screenshot of how they look on your localhost, and I'll see if I'm the one not noticing any differences.

**Review by Njong392 — COMMENTED** [review]
Thank you for these changes! Do check the reviews I have made.

File: src/components/navbar.jsx (hunk 2 above)
Comment (hassancodess) [reply src/components/navbar.jsx:14]: I have made a new commit. Check it now!

File: src/components/form.jsx (hunk 1 above)
Comment (Njong392) [inline src/components/form.jsx:88]: How about you change the hover color of the submit button too? What do you think?

File: src/components/navbar.jsx [hunk 3]
```diff
@@ -10,8 +10,8 @@ const Navbar = () => {
...
```
Comment (Njong392) [inline src/components/navbar.jsx:14]: I think the `Get Started` button should not have the hover effect yet. When the user hovers, the contrast between the active link and the new hover effect you added does not look very good. While we are still trying to fix the contrast issues, I'll suggest you not add new background color on hover for now.

**Review by Njong392 — CHANGES_REQUESTED** [review]

**Comment by Njong392** [comment]
Hello @hassancodess how is this coming along?

**Comment by Njong392** [comment]
Hello @hassancodess are you still up for working on this PR? If it's stagnating, I'm sorry but I'll have to close it.

**Pull request closed without merging**